.github/workflows/release-e2e-workflow-template.yml

```diff
@@ -127,7 +139,11 @@ jobs:
       - run: npx cypress cache path
       - name: Cypress tests
         uses: cypress-io/github-action@v2
+        env:
+          # remove success video to save space and memory
+          CYPRESS_DELETE_SUCCESS_VIDEOS: true
```

Comment:
Instead of record then delete videos, just disable recording:
https://github.com/opensearch-project/opensearch-dashboards-functional-test/blob/main/integtest.sh#L219-L223

Reply:
Yes, but we want to keep the video if the specs failed.

Reply:
And even if we downgrade chromium version to 112, the security test will still fail. I am trying to reduce the memory our test cases are using.

Reply:
Got it.